string human_id
geometry_msgs/Point center_of_attention
sensor_msgs/PointCloud2[] elements